charts/external-dns/templates/deployment.yaml

```
@@ -88,7 +87,6 @@ spec:
             {{- if and (eq .Values.txtPrefix "") (ne .Values.txtSuffix "") }}
```

```suggestion
            {{- if and (not .Values.txtPrefix "") .Values.txtSuffix }}
```
This change would be worth making for consistency while modifying this area.